tests/dag_processing/test_processor.py

```diff
@@ -231,6 +233,7 @@ def test_dag_file_processor_sla_miss_doesnot_raise_integrity_error(self, dag_mak
             .count()
         )
         assert sla_miss_count == 1
+        mock_stats_incr.assert_called_with('sla_missed')
```

Comment:
Yes I thought about this but I choose this test on purpose as this is one of the only ones(if not the only one) that creates the SLA by the scheduler code (instead of creating it inside the testing code). \
Therefore I feel like this was the best way to do it.